Assertion with double click in PyColourChooser

**Operating system**:  Windows 10, 64 bit
**wxPython version & source**: pypi `wx.version: 4.1.0 msw (phoenix) wxWidgets 3.1.4`
**Python version & source**: stock 3.6.4, 32 bit

**Description of the problem**: When double clicking in the colour chooser, an assertion is raised.
See screenshot below for the result of running demo/PyColourChooser.py
I have double-clicked inside the colour gradient rectangle (as still being indicated by the small circle.

I did not run the demo in the debugger, but wxGlade and there the assertion is raised in wx.lib.pycolourchooser:
```
pycolourchooser.py,  line 474, in onPaletteUp
  self.palette.ReleaseMouse() # Must call once for each CaputreMouse()
```
The error message is:
```
wx._core.wxAssertionError: C++ assertion ""!wxMouseCapture::stack.empty()"" failed at ..\..\src\common\wincmn.cpp(3370) in wxWindowBase::ReleaseMouse(): Releasing mouse capture but capture stack empty?
```

![PyColourChooser](https://user-images.githubusercontent.com/5512021/84809203-05b30000-b00a-11ea-9609-2bd83509623c.png)

--- a/wx/lib/colourchooser/pycolourchooser.py
+++ b/wx/lib/colourchooser/pycolourchooser.py
@@ -453,7 +453,8 @@ class PyColourChooser(wx.Panel):
         self.colour_slider.CaptureMouse()
 
     def onSliderUp(self, event):
-        self.colour_slider.ReleaseMouse()
+        if self.colour_slider.HasCapture():
+            self.colour_slider.ReleaseMouse()
 
     def onSliderMotion(self, event):
         """Handle mouse-down drag on the colour slider palette"""
@@ -471,7 +472,8 @@ class PyColourChooser(wx.Panel):
 
     def onPaletteUp(self, event):
         """Stores state that the mouse is no longer depressed."""
-        self.palette.ReleaseMouse() # Must call once for each CaputreMouse()
+        if self.palette.HasCapture():
+            self.palette.ReleaseMouse() # Must call once for each CaputreMouse()
 
     def onPaletteMotion(self, event):
         """Updates the colour values during mouse motion while the

